src/test/streams_tests.cpp

```diff
@@ -9,8 +9,7 @@
 #include <boost/assign/std/vector.hpp> // for 'operator+=()'
 #include <boost/assert.hpp>
 #include <boost/test/unit_test.hpp>
-                    
-using namespace std;
+
 using namespace boost::assign; // bring 'operator+=()' into scope
```

Comment:
Actually, I don't think I'll fix it as it would no doubt break the same-binaries condition I've been working hard to retain. With a big update I think upholding that condition is worthwhile.